tests/ct_connection.c

```diff
@@ -13,26 +13,47 @@
 #define CLUSTER_USERNAME "default"
 #define CLUSTER_PASSWORD "secretword"
 
+int connect_success_counter;
+int connect_failure_counter;
+void connect_callback(const redisContext *c, int status) {
+    (void)c;
+    //fprintf(stderr, "Connect to %s:%d %s\n", c->tcp.host, c->tcp.port, status == REDIS_OK ? "OK" : "ERR");
```

Comment:
Is this good-to-have printouts ?

Reply:
Oops, it's a "was good to have" but not anymore